frontend/src/console/components/Hero/Hero.tsx

```diff
@@ -0,0 +1,287 @@
+
+    this.setState({
+      [field]: value
+    });
+  };
+
+  private updateHeroText = () => {
+    if (this.activeInstanceDataExists()) {
+      const homepageOverlayHtml = `<h1>${this.state.title}</h1><p>${this.state.subtitle}</p>`;
+      this.props.updateStaticContentOverridesFieldValue(
+        this.props.activeInstance.data!.id,
+        'homepageOverlayHtml',
+        homepageOverlayHtml
+      );
+    }
+  };
+
+  private onChangeColor = (fieldName: string, newColor: string) => {
+    if (this.activeInstanceDataExists()) {
+      this.props.updateThemeFieldValue(
+        this.props.activeInstance.data!.id,
+        fieldName,
+        newColor
+      );
+    }
+  };
+
+  private updateImage = (imageName: string, image: File) => {
+    if (this.activeInstanceDataExists()) {
+      this.props.updateImages(
+        this.props.activeInstance.data!.id,
+        imageName,
+        image
+      );
+    }
+  };
+
+  private removeImage = (imageName: string) => {
+    if (this.activeInstanceDataExists()) {
+      this.props.updateImages(
+        this.props.activeInstance.data!.id,
+        imageName,
+        ''
+      );
+    }
+  };
+
+  public render() {
+    const instance = this.props.activeInstance;
+    let themeData: undefined | ThemeSchema;
+    let staticContentOverrides: undefined | StaticContentOverrides;
+
+    if (this.themeConfigExists()) {
+      themeData = instance.data!.draftThemeConfig;
+    }
+
+    if (this.staticContentOverridesExists()) {
+      staticContentOverrides = instance.data!.draftStaticContentOverrides;
+    }
+    return (
+      <div className="hero-page">
+        <ConsolePage contentLoading={this.props.loading}>
+          <ConsolePageCustomizationContainer>
+            <Row>
+              <Col md={9}>
+                <h2>
+                  <WrappedMessage id="hero" messages={messages} />
+                </h2>
+                <p>
+                  <WrappedMessage id="heroDescription" messages={messages} />
+                </p>
+              </Col>
+            </Row>
+            {themeData && themeData.version === 1 && (
+              <Row>
+                <Col md={12}>
+                  {this.themeConfigExists() &&
+                    this.staticContentOverridesExists() && (
+                      <HeroPreview
+                        heroCoverImage={instance.data!.heroCoverImage || ''}
+                        homePageHeroTitleColor={
+                          themeData.homePageHeroTitleColor
+                        }
+                        homePageHeroSubtitleColor={
+                          themeData!.homePageHeroSubtitleColor
+                        }
+                        homepageOverlayHtml={
+                          staticContentOverrides!.homepageOverlayHtml
+                        }
+                      />
+                    )}
+                </Col>
+              </Row>
+            )}
+            <CollapseEditArea initialExpanded>
```

Comment:
@Agrendalath, I added it because there are 2 separate sections/components in the page - the preview component and the fields to edit the data and it made sense to add the latter under a `CollapseEditArea`.

Reply:
@Agrendalath, I will leave it as-is unless this is a dealbreaker. We can always change this later if required. CC @giovannicimolin

Reply:
@lgp171188, sounds good to me :+1: